### DISCUSSION TITLE
About mmcv and mmdet version

### DISCUSSION THREAD

**Discussion by gly2000** [ethvis/qd-track#20] [Q&A]
What is mmcv and mmdet version? 
Import Error: cannot import name 'wrap_fp16_model'
Maybe the version is wrong. Thank you.

**Comment by OceanPang** [comment]
Hi @gly2000 , this issue is already fixed in #18 .

**Reply by gly2000** [reply]
When I use the newest mmcv and mmdet, there is a new error.
AttributeError：‘ConfigDict’ object has no attribute 'test_cfg'
How to solve it: Thank you.
